docs/RELEASE-NOTES.rst

```diff
@@ -7,7 +7,24 @@ Release Notes for BIG-IP Controller for Kubernetes
 Added Functionality
 ```````````````````
 * Added route domain support for the VxLAN forwarding data base addresses (FDB).
-  Also, Removed the restriction on setting the default route domain
+  Also, removed the restriction on setting the default route domain
+* Flannel VxLAN support. You can now run the controller in Kubernetes in cluster mode
+  with a Flannel VXLAN tunnel specified, and the controller will populate FDB and ARP entries
+  on the BIG-IP.
+* Ingresses will now share a virtual server if specified with the same IP address and port.
```

Comment:
I think you should have @jputrino look at these to check for grammar, voice, etc. Seems like it could be simplified a bit.